### ISSUE TITLE
Crashes a lot when trying to use modules

### ISSUE THREAD

**Issue by Chronoxzc** [Dino-log/AidsHack#1]

**Comment by Dino-log** [comment]
yes, this client is discontinued... i was mid fixing this verison when i just lost motivation...
use [MarlowClient](https://github.com/MarlowDevelopment/MarlowClient/)
